src/ansible_builder/containerfile.py

```diff
@@ -290,15 +301,15 @@ def _handle_additional_build_files(self):
             ee_file = Path(self.definition.filename)
             if src.is_absolute():
                 if not src.exists():
-                    logger.warning(f"User build file {src} does not exist.")
+                    logger.warning("User build file %s does not exist.", src)
```

Comment:
This and the other logging changes seems to be an unrelated change. FWIW, I'm fine with this format, but we should at least enable the pylint logging checker and the checks for logging string format (W1202, W1203) so that we are consistent. Perhaps that should be a separate change, but fine if you want to do it here.